<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_strecke</class>
 <widget class="QDialog" name="Dialog_strecke">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_epx"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_spx">
      <property name="text">
       <string>Startpunkt X</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_spx"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_spy">
      <property name="text">
       <string>Startpunkt Y</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_spy"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_spz"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_epy"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_epz"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_spz">
      <property name="text">
       <string>Startpunkt Z</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_epx">
      <property name="text">
       <string>Endpunkt X</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_epy">
      <property name="text">
       <string>Endpunkt Y</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_epz">
      <property name="text">
       <string>Endpunkt Z</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_spx</tabstop>
  <tabstop>lineEdit_spy</tabstop>
  <tabstop>lineEdit_spz</tabstop>
  <tabstop>lineEdit_epx</tabstop>
  <tabstop>lineEdit_epy</tabstop>
  <tabstop>lineEdit_epz</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_abbrechen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
